lib/child_process.js

```diff
@@ -266,10 +266,10 @@ exports.execFile = function(file /*, args, options, callback*/) {
     }
 
     if (args.length !== 0)
-      cmd += ' ' + args.join(' ');
+      cmd += ` ${args.join(' ')}`;
 
     if (!ex) {
-      ex = new Error('Command failed: ' + cmd + '\n' + stderr);
+      ex = new Error(`Command failed: ${cmd}\n${stderr}`);
```

Comment:
Likewise here, it's not likely worth changing the string concatenations in error throws since those will be changing very soon.